app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -103,6 +114,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         binding = DataBindingUtil.inflate(inflater, R.layout.fragment_attendees, container, false);
 
         binding.setAttendees(attendeesPresenter.getAttendees());
+        setRetainInstance(true);
```

Comment:
![device-2017-06-22-003731](https://user-images.githubusercontent.com/3874064/27402073-07eaaa2c-56e3-11e7-9a9e-da6019c5a659.png)

No, it's not